compiler/fir/fir-serialization/src/org/jetbrains/kotlin/fir/serialization/FirElementSerializer.kt

```diff
@@ -199,8 +198,8 @@ class FirElementSerializer private constructor(
         var hasGetter = false
         var hasSetter = false
 
-        //val compileTimeConstant = property.compileTimeInitializer
```

Comment:
@mglukhikh It looks like FE 1.0 actually has a field in property descriptor and initializes it during field value resolution if the property has a constant. I think I found a counter part in `fir`, but I'm not sure we want to add `compileTimeInitializer` (of `ConstantValue` type perhaps) to `FirProperty` or just determining whether it has constant or not during serialization like this.